cranelift/filetests/filetests/isa/x64/wasm/load_store_dynamic_kind_i32_index_0_guard_no_spectre_i32_access_0_offset.wat

```diff
@@ -71,14 +71,14 @@
 ;;   movabsq $-4, %r11
 ;;   addq    %r11, 8(%rsi), %r11
 ;;   cmpq    %r11, %r10
-;;   jbe     label1; j label3
-;; block1:
-;;   movq    0(%rsi), %rsi
-;;   movl    0(%rsi,%r10,1), %eax
-;;   jmp     label2
+;;   jnbe    label1; j label2
```

Comment:
This PR introduced a lot of noise for cases where a `brz` was replaced with a `brif` with the branch order reversed. This change is equivalent in behavior to the previous output, but ends up changing the label names.